docs/JLBP-16.md

```diff
@@ -1,27 +1,37 @@
-# [JLBP-16] Ensure upper version alignment of dependencies for consumers
+# [JLBP-16] Ensure upper version alignment of dependencies
 
 For multi-module projects, this best practice assumes you have already applied
 [JLBP-15](JLBP-15.md), so that your project has a BOM.
 
-Upper version alignment is defined in the [glossary](glossary.md).
+Upper version alignment means that when more than one version of a dependency
+is present in a module's dependency tree, the build system selects
+the highest version. Upper version alignment ensures that when packages are
+upgraded to higher versions which don’t make any breaking changes,
+there are no new linkage conflicts.
 
 ## Achieving upper version alignment
 
 Upper version alignment increases the likelihood that consumers' build systems
 select the right versions of direct and transitive dependencies, reducing the
 number of conflicts.
 
-As noted in the definition, to fix a version misalignment caused by a shorter
-path to a version of a dependency that is not the upper bound in the
-dependency tree, a direct dependency needs to be added so that Maven consumers
-select the correct version.
+Gradle selects the highest version automatically. Maven, however, selects
+the version it encounters first when traversing the dependency tree, 
+regardless of version. 
 
-See the details specific to each build system in the following sections.
+To fix a version misalignment caused by a shorter
+path to a lower version of a dependency, you can either:
 
-## Maven
+1. Update the dependency on the lower version to the upper bound.
+   This usually requires submitting a patch to another project 
+   and waiting for it to release. 
+   
+2. Add a direct dependency on the misaligned dependency to your 
+   own project, even though your project does not directly 
+   import any of that dependency's classes. 
```

Comment:
`dependencyManagement` section capability is suitable for this purpose. I feel that's better than declaring an unused dependency.